src/test/java/NumberQuestionTest.java

```diff
@@ -0,0 +1,83 @@
+
+import org.junit.Before;
+import org.junit.Test;
+import survey.NumberQuestion;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class NumberQuestionTest {
+    NumberQuestion nq1;
+    NumberQuestion nq2;
+    NumberQuestion nq3;
+    NumberQuestion nq4;
+
+    @Before
```

Comment:
FYI, given the style here, I think this should be a BeforeEach. Cause you don't want tests to be affected when you make changes.

Reply:
What do you mean? Does "before" affect tests when you make changes?

Reply:
Discussed offline, its good to merge.